samples/highcharts/data-tools/datapool-json-connector/demo.js

```diff
@@ -0,0 +1,139 @@
+Dashboards.board('container', {
+    dataPool: {
+        connectors: [{
+            type: 'JSON',
+            id: 'array-of-objects',
+            options: {
+                data: [{
+                    name: 'John',
+                    age: 23,
+                    weight: 70
+                }, {
+                    name: 'Jane',
+                    age: 25,
+                    weight: 65
+                }, {
+                    name: 'Joe',
+                    age: 21,
+                    weight: 68
+                }, {
+                    name: 'Jack',
+                    age: 26,
+                    weight: 72
+                }]
+            }
+        }, {
+            type: 'JSON',
+            id: 'to-parse',
+            options: {
+                orientation: 'columns',
+                beforeParse: function (data) {
+                    const res = [],
+                        response = data[0].series;
+                    response.forEach(series => {
+                        res.push([series.name].concat(series.data));
+                    });
+
+                    return res;
+                },
+                data: [{
+                    series: [{
+                        name: 'Series 1',
+                        data: [1, 2, 3, 4]
+                    }, {
+                        name: 'Series 2',
+                        data: [2, 3, 4, 5]
+                    }]
+                }]
+            }
+        }, {
+            type: 'JSON',
+            id: 'no-first-names',
+            options: {
+                firstRowAsNames: false,
+                columnNames: ['random name', 'second random name', 'third random name'],
+                data: [
+                    [1, 2, 3],
+                    [2, 3, 4]
+                ]
+            }
+        }, {
+            type: 'JSON',
+            id: 'rows',
+            options: {
+                orientation: 'rows',
```

Comment:
The `rows` is the default value of the `orientation` param, the declaration is unnecessary here.